src/components/Layout/LeftSidebar.tsx

```diff
@@ -176,8 +208,8 @@ const LeftSidebar = ({ inHeader = false }: LeftSidebarProps) => {
                   size="12px"
                 />
               </Accordion.Trigger>
-              <Accordion.Content className={cn(accordionContentClassName, 'px-2 py-3')}>
-                <ChildAccordion content={productObj.nav.content} layer={0} tree={[index]} />
+              <Accordion.Content className={accordionContentClassName}>
+                <ChildAccordion content={[...productObj.nav.content, ...productObj.nav.api]} tree={[index]} />
```

Comment:
With this and the surrounding logic, we consider content and api references as part of one collection, meaning that API references are integrated within the main nav.